src/ResourceManager/MachineLearningCompute/AzureRM.MachineLearningCompute.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.2.0'
+ModuleVersion = '0.3.0'
```

Comment:
@shutchings you can revert the changes in this PR that bump the module version - we will handle that with separate tooling

Reply:
Done.